kafkajobs/driver.go

```diff
@@ -115,6 +115,16 @@ func FromConfig(tracer *sdktrace.TracerProvider, configKey string, log *zap.Logg
 		return nil, errors.E(op, err)
 	}
 
+	pingCtx, pingCancel := context.WithTimeout(context.Background(), conf.Ping.Timeout)
+	defer pingCancel()
+
+	err = jb.kafkaClient.Ping(pingCtx)
+	if err != nil {
+		return nil, errors.E(op, errors.Errorf("ping kafka was failed: %s", err))
+	}
+
+	log.Debug("ping kafka: ok", zap.String("driver", pipeline.Driver()), zap.String("pipeline", pipeline.Name()))
+
```

Comment:
Just to DRY, you can move these 2 segments into the one method:
```go
func ping(kafkaClient, timeout) error {
    pingCtx, pingCancel := context.WithTimeout(context.Background(), timeout)
	defer pingCancel()

	err = kafkaClient.Ping(pingCtx)
	if err != nil {
		return nil, errors.E(op, errors.Errorf("ping kafka was failed: %s", err))
	}

	log.Debug("ping kafka: ok", zap.String("driver", pipeline.Driver()), zap.String("pipeline", pipeline.Name()))
    return nil
}

Reply:
I wanted to do this, but I was scared by the number of parameters in the function 😄